### ISSUE TITLE
Not working anymore

### ISSUE THREAD

**Issue by davyfreeman** [Shenmue-Mods/Shenmue12launcher#1]
Hi,
I've come here through Sappharad's post on shenmuedojo.com.
I suppose the fix is not working anymore, unless I made an error while applying it.
That is to say, it still opens up the launcher.
Since their last post on there, there have been a few updates to the game.

**Comment by Sappharad** [comment]
I haven’t heard of any reports of the launcher changing with updates, they don’t really have a reason to change it. I’ll confirm in about 14 hours.

**Comment by Sappharad** [comment]
It still works fine, I just tested it against the current version of the game. And I re-analyzed the original launcher, nothing has changed in the launcher that would break this version.

You're doing something wrong. Closing this issue.

An easy way to test is to install the modded launcher over the original, go into Steam properties for the game and click Advanced Launch arguments. Add "sm2" as your advanced argument, then try launching the game. It will launch directly into Shenmue 2, bypassing the launcher. Then just go back into properties and clear out the argument to revert back to default behavior.
